src/components/Chat/constants.ts

```diff
@@ -21,12 +21,13 @@ export const AVATAR_DUMMY_IMAGE = [
 
 export const TWITTER_SHARE_CONTENT = (
   wizardName: string,
-  url: string
-) => `Check this out! I'm talking to this DeAI agent ${wizardName} built by ELNA.ai
+  url: string,
+  user_X_Handle: string
+) => `Check this out! I'm talking to DeAI agent ${wizardName} built ${user_X_Handle ? `by @${user_X_Handle} on` : "by"} ELNA.ai
```

Comment:
```suggestion
) => `Check this out! I'm talking to DeAI agent ${wizardName} built ${user_X_Handle ? `by @${user_X_Handle} on` : "on"} ELNA.ai
```

Reply:
It was 'by' earlier, used 'on' when the creator handle is used and kept by for the default case just in case you guys preferred that, anyways updated it and everything else in https://github.com/elna-ai/ELNA-DApp/pull/213/commits/ee62160ed7a5e237a05a849341084f0f1525932d